pkgs/by-name/wh/whitesur-gtk-theme/package.nix

```diff
@@ -28,17 +28,18 @@ let
   single = x: lib.optional (x != null) x;
 
 in
-lib.checkListOfEnum "${pname}: window control buttons variants" [ "normal" "alt" "all" ] altVariants
+lib.checkListOfEnum "whitesur-gtk-theme: window control buttons variants" [ "normal" "alt" "all" ]
```

Comment:
Is `pname` unsuitable for `checkListOfEnum`? Using `pname` seems reasonable for that purpose. Even if the repository name differs, displaying `pname` is still useful.